### DISCUSSION TITLE
Unable to compile the project

### DISCUSSION THREAD

**Discussion by Arslankhan123** [technologiestiftung/giessdenkiez-de#314] [General]
I am trying to compile the project on my local machine. I have a basic knowledge in Python and struggling to start the compilation. 

Can someone please help me to write the command to start the project. 

I am trying the run npm start but it is complaining me to setup BUILD_TARGET and NODE_ENV which I set like 

**npm start BUILD_TARGET=demo NODE_ENV=test**

It is giving me the error 

cross-env BUILD_TARGET=DEFAULT NODE_ENV=development webpack-dev-server --config webpack/dev.config.js "BUILD_TARGET=demo" "NODE_ENV=test"

Can you please tell me what I am missing?
I have windows machine with npm version 6.14.12 and Pyhon version 3.9.3

**Comment by vogelino** [comment]
Hi! 

My first guess is that you should declare the environment variable before running ˋnpm run startˋ. But the easiest way would be to create an ˋ.envˋ file (see our ˋ.env.exampleˋ).

We are currently writing more complete documentation on how to setup your own Gieß Den Kiez. Have a look at our repo-wiki. Be aware that this is a work in progress. We gladly accept any feedback regarding our documentation.

We also host a workshop in which we will try to give a big overview of all the moving parts of the application. It will be in German but if you're interested, be sure to register!

https://hopin.com/events/rezept-wie-baue-ich-mir-giess-den-kiez

We will try to find a way to translate it later on and we'll make sure to keep you posted. 

I hope this helps. Cheers!
